azkaban-webserver/src/main/resources/azkaban/webapp/servlet/velocity/executingflowpage.vm

```diff
@@ -112,6 +112,9 @@
         <li class="nav-button pull-right"><button type="button" id="cancelbtn" class="btn btn-danger btn-sm">Kill</button></li>
         <li class="nav-button pull-right"><button type="button" id="retrybtn" class="btn btn-success btn-sm">Retry Failed</button></li>
         <li class="nav-button pull-right"><button type="button" id="executebtn" class="btn btn-success btn-sm">Prepare Execution</button></li>
+        #if ($executionExternalLinkURL)
+          <li class="nav-button pull-right"><a id="analyzerButton" href="${executionExternalLinkURL}" class="btn btn-info btn-sm" type="button" target="_blank" title="Analyze job in ${executionExternalLinkLabel}" tabindex="-1">${executionExternalLinkLabel}</a></li>
```

Comment:
Currently there is a bug in the page with the natural tab order. Consider the following case.

'Pause' and 'Resume' buttons are displayed on the page. The 'li' element 'Pause' appears before 'Resume' and hence takes the right most spot. So in the page we will see as follows.
[Resume] | [Pause]

Let's consider the tab order now. Let's say tab focus is on 'Stats'. When user presses tab again we will expect it to go to 'Resume' as that's the first button on the right. But tab focus goes to 'Pause' first and then to 'Resume' which will be annoying.

Now, when Dr. Elephant is enabled this button will always be displayed and the tab order is broken which is not nice. I didn't find any elegant fix for this as of now. Hence, I have disabled tab focus on this button by setting tabindex="-1"

Hope this explains :)